org/branchprotection.yml

```diff
@@ -195,3 +195,21 @@ branch-protection:
           protect: false
         buildpacks-envs:
           protect: false
+
+        cnbapplifecycle:
+          allow_deletions: false
+          allow_disabled_policies: true
+          allow_force_pushes: false
+          enforce_admins: true
+          include: ["^main$"]
+          protect: true
+          required_status_checks:
+            contexts:
+              - "Go / unit (pull_request)"
+              - "EasyCLA"
+          required_pull_request_reviews:
+            bypass_pull_request_allowances:
+              teams: ["wg-app-runtime-platform-bots"]
```

Comment:
Will the proposed change enforce bypass only for `wg-app-runtime-platform-bots` across all repos, including other workgroups?